Packs/CheckPointHEC/Integrations/CheckPointHEC/CheckPointHEC.py

```diff
@@ -97,9 +97,17 @@ def test_module(client: Client):
     demisto.results('ok')
 
 
-def fetch_incidents(client: Client):
+def fetch_incidents(client: Client, first_fetch: str):
+    def get_timedelta() -> timedelta:
```

Comment:
Good work!
You can use the `parse_date_range` function to convert the `first_fetch` to a time.
see [here](https://github.com/demisto/content/blob/master/Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py#L1254).